### ISSUE TITLE
Method <type> failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#130]
```
Operating system and version (32/64bit) : windows 7 32
.Net Framework version : 4
Office name and version(32/64bit) : office 2010 32
Browser name and version : firefox 35.0
SeleniumWrapper version : 1.0.23.0

What steps will reproduce the problem with a public website ?

I have a button in a Access form to call a site.
I replaced my code for the following
   Dim selenium As New SeleniumWrapper.WebDriver
   selenium.start "firefox", "http://www.google.com/"
   selenium.Open "/"
   selenium.Type "name=q", "Eiffel tower"
   selenium.Click "name=btnG"
I got the same error


What is the expected output? What do you see instead?

When i press the button to call the site it shows firefox and the site but it 
doesn't type Eiffel tower in the box.
Behind firefox there is a popup window that says:
Run-time error '-2146232832 (80131600)'
Method <type> failed!
NullReferenceException: Object reference not set to an instance of an object.

Please provide any additional information below.


```

Original issue reported on code.google.com by `pemif...@gmail.com` on 16 Jan 2015 at 5:34

Attachments:
- [error.jpg](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-130/comment-0/error.jpg)

**Comment by GoogleCodeExporter** [comment]
```
This error was introduced by Firefox 35.
So until it's fixed in the Selenium framework, you can downgrade do Firefox 34
or use the Selenium 2 command:
selenium.findElementByName("q").SendKeys "Eiffel tower"
```

Original comment by `florentbr` on 18 Jan 2015 at 12:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Worked for me!
Thank you!
```

Original comment by `pemif...@gmail.com` on 6 Mar 2015 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 2.0.2.0
http://florentbr.github.io/SeleniumBasic/
```

Original comment by `florentbr` on 22 Jul 2015 at 4:21
- Changed state: **Fixed**
